network-mapper/templates/agent-daemonset.yaml

```diff
@@ -122,13 +122,21 @@ spec:
           - name: OTTERIZE_HOST_PROC_DIR
             value: /host/proc
 
-          {{ if .Values.global.azure.enabled }}
```

Comment:
what happened here? Seems like an unintentional deletion

Reply:
not unintentional, but I'll put it in another PR